.github/workflows/yscope-velox-builder-img.yml

```
@@ -0,0 +1,110 @@
+            echo "exists=true" >> "$GITHUB_OUTPUT"
+            echo "Image already exists, skipping build"
+          else
+            echo "exists=false" >> "$GITHUB_OUTPUT"
+            echo "Image does not exist, will build"
+          fi
+
+      - name: Extract GitHub metadata
+        if: steps.check-image.outputs.exists == 'false'
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{steps.image-tag.outputs.tag}}
```

actionlint (1.7.10)

[error] property "image-tag" is not defined in object type {check-image: {conclusion: string; outcome: string; outputs: {string => string}}; deps-hash: {conclusion: string; outcome: string; outputs: {string => string}}; image-ref: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)